### PR TITLE
Export Admin Facade

### PR DESCRIPTION
**Pull request by SimonRichardson**
## Description of change

The following outputs the Admin facade so it makes it possible to login
via another source, other than using the local credentials.

The code is quite simple as we fall back to the facade.Type rather than
asking the registry for the information. We have to do this, as the
Admin facade details are outside of the registry and I'm not sure why
this is...

## QA steps

```sh
make rebuild-schema
```

## Bug reference

Fixes: https://bugs.launchpad.net/juju/+bug/1885324

### CODE REVIEW

File: apiserver/facades/schema.json [hunk 1]
```diff
@@ -848,6 +848,267 @@
             }
         }
     },
+    {
+        "Name": "Admin",
+        "Description": "admin is the only object that unlogged-in clients can access. It holds any\nmethods that are needed to log in.",
```
Comment (SimonRichardson) [inline apiserver/facades/schema.json:853]: I don't believe this is true anymore, we should update the description.

Comment (wallyworld) [reply apiserver/facades/schema.json:853]: CMR callers can access various facades, but they do need a valid macaroon to get past auth

**Comment by SimonRichardson** [comment]
$$merge$$

**Comment by SimonRichardson** [comment]
https://trello.com/c/qwHY8gdk/2739-bug-1885324-export-the-admin-facade-in-schema

**Pull request merged**